packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -409,7 +409,9 @@ describe("renderer", () => {
             markImagesAsLoaded();
 
             // Assert
-            const imageNodes = screen.queryAllByRole("img");
+            const imageNodes = screen.queryAllByAltText(
```

Comment:
Sadly we can't query for img roles anymore, so I've updated all of these to use the appropriate alttext.